lib/tasks/licenses.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+namespace :licenses do
+
+  report_path = Rails.root.join('doc/licenses/licenses.xml')
+  temp_report_path = Rails.root.join('doc/licenses/temp_licenses.xml')
+
+  desc 'Generates a report with the dependencies and their licenses'
+  task :report do
+    $stdout.puts 'Generating report...'
+    exec("license_finder report --format=xml --save=#{report_path}")
+  end
+
+  desc 'Check license compliance of dependencies'
+  task :compliance do
+    $stdout.puts 'Checking license report is up to date...'
+
+    exec "license_finder report --format=xml --save #{temp_report_path}"
+
+    unless system("diff #{report_path} #{temp_ci_report_path} --brief")
+      $stderr.puts '*** License report outdated. Please generate again. ***' # rubocop:disable Style/StderrPuts
+      exit 1
+    end
+
+    $stdout.puts 'Checking license compliance...'
```

Comment:
`$stderr` everywhere.

Reply:
WDYM? `stderr` is to put when the task fails, or is it?